### DISCUSSION TITLE
eQSL column in logbook view

### DISCUSSION THREAD

**Discussion by sebket** [magicbug/Cloudlog#1208] [General]
Hi, first of all, thanks for the great work you're doing! Donation will follow.

I like a lot how the status of sent/received qsl cards is displayed in the logbook view.

As I mostly do digital modes I give/get most confirmations via eQSL. Therefore it'd be great if there'd be a column to show the status of eQSL in just the same way as for paper qsl, with the up and down arrows.

Best,
Sebastian DO2KSL

**Comment by dg9vh** [comment]
[image: image.png]
There is already such a display. What do you need more?
73 de Kim
DG9VH

Am Fr., 1. Okt. 2021 um 21:43 Uhr schrieb sebket ***@***.***>:

> Hi, first of all, thanks for the great work you're doing! Donation will
> follow.
>
> I like a lot how the status of sent/received qsl cards is displayed in the
> logbook view.
>
> As I mostly do digital modes I give/get most confirmations via eQSL.
> Therefore it'd be great if there'd be a column to show the status of eQSL
> in just the same way as for paper qsl, with the up and down arrows.
>
> Best,
> Sebastian DO2KSL
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/magicbug/Cloudlog/discussions/1208>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADKN5OQROK7DB3IH2S6N6NTUEYFNPANCNFSM5FFL7J7A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Amateurfunk ist das, was uns verbindet! Bleiben wir in Kontakt!
Ortsverband Q03, Neunkirchen

**Reply by sebket** [reply]
Kim,
thank you for reply.
If there is, I wasn't able to find it.
I only get a column called QSL and the up/down arrows show only the status of paper qsl.
If a qso was confirmed by eQSL only, the arrows stay red.

73 Sebastian DO2KSL

**Reply by magicbug** [reply]
Should show up regardless of paper being selected as long as eQSL infos been provided the eQSL columns shown and you can click the incoming arrow and it will display the card as per the info in the Wiki https://github.com/magicbug/Cloudlog/wiki/eQSL

Make sure your running the latest code from Github.

**Reply by sebket** [reply]
I think that must be it then. The eQSL status in my qso entries was set "manually" via adif import. I did not set up the connection to my eQSL account in cloudlog.
I'll change that and I'm sure it'll work fine afterwards. Thanks for the help!
73 Sebastian
